Makefile

```diff
@@ -98,6 +98,7 @@ SHELLCHECK_VER ?= v0.7.2
 BUILD_TIMESTAMP_W_SEC := $(shell date +%Y-%m-%d-%H-%M-%S)
 EKS_CLUSTER_NAME := integ-cluster-$(BUILD_TIMESTAMP_W_SEC)
 AWS_REGION := us-west-2
+AWS_TEST_UUID := $(shell openssl rand -hex 6) 
```

Comment:
Why don't we set this variable in the bats test suite? Similar to

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/8a9ccd19fee03495e4b43f99ac10198bbfdf948c/test/bats/azure.bats#L19-L21

I think we should avoid having all these defaults in the `Makefile` and keep them in the script file or bats test file instead.

Reply:
I initially set this variable in the bats test suite  - https://github.com/lasred/secrets-store-csi-driver/blob/master/test/bats/aws.bats#L13 

but the issue is that with bats, this file gets sourced multiple times(in setup, and per every test) so I had conflicting UUIDs in setup_file where I created the secrets and test "deploy aws secretproviderclass crd" where the UUID was used to create the secret provider class

So I opted for the variable to be defined in the Makefile and passed into the bats script